tests/__init__.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+
+from flask import Blueprint
+from flask_testing import TestCase
+
+os.environ["BOT_API_KEY"] = "abcdefg"  # This is a constant, must be done first
+os.environ["PAPERTRAIL_ADDRESS"] = 'localhost'  # satisfies coverage
+os.environ["DATADOG_ADDRESS"] = 'localhost'  # satisfies coverage
+try:
+    del os.environ["FLASK_DEBUG"]  # Some unit tests fail if this is set
+except KeyError:
+    pass
```

Comment:
Probably just me, but I think just checking whether it exists would be a fair bit cleaner.
```py
if "FLASK_DEBUG" in os.environ:
    del os.environ["FLASK_DEBUG"]  # Some unit tests fail if this is set
```

Reply:
good point!